src/bolusScreen.js

```diff
@@ -8,20 +8,47 @@ class BolusScreen {
     async Open() {
         try {
             //assume we are starting from the open screen
-            await match.accessible.Button(BolusLabel.Bolus).tap();
+            await match.accessible.Button(text.bolusScreen.Bolus).tap();
```

Comment:
[nit] For all of these `Open` and `Close` functions for the various screens, is there a way we can ensure that it actually opened or closed properly and ended up in the final expected location? I see here we are waiting until the bolus button is tapped, but we haven't actually confirmed that we opened the bolus screen itself. Or is that confirmation part of the tests that call the `Open` and `Close`? To me it would make sense to confirm here so we know if `Open` or `Close` returned successfully then we can be certain we are in the expected state. Thoughts?

Reply:
i could open and then assert I can see the screens header